ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1066,3 +1066,10 @@ literals:
 
 - min_zone_air_relative_humidity_sensor
 - min_zone_air_relative_humidity_setpoint
+
+## New fields for air quality sensors
+- thirtysecondrolling_average_zone_air_co_concentration_sensor
+- thirtysecondrolling_average_zone_air_particle_diameter_sensor
+- zone_air_no2_probability_percentage_sensor
```

Comment:
and here.  I note that this one doesn't show the internal NO2 sensors.  Is that an omission or are these devices just different?

Reply:
this is the `fields.yaml` file, not a type definition. is this comment regarding the field itself?